### PR TITLE
Add vanilla pg baseline tests

### PR DESCRIPTION
**Pull request by bojanserafimov**
https://github.com/zenithdb/zenith/issues/553

### CODE REVIEW

File: test_runner/fixtures/compare_fixtures.py [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+from abc import ABC, abstractmethod
+
+from fixtures.zenith_fixtures import PgBin, PgProtocol, VanillaPostgres, ZenithEnv
+from fixtures.benchmark_fixture import MetricReport, ZenithBenchmarker
+
+# Type-related stuff
+from typing import Iterator
+
+
+class PgCompare(ABC):
```
Comment (hlinnaka) [inline test_runner/fixtures/compare_fixtures.py:12]: A bigger comment block would be nice, explaining what PgCompare is for, how to use it, etc. Either here, or at the top of the file.

Comment (hlinnaka) [reply test_runner/fixtures/compare_fixtures.py:12]: Oh, I see there's a comment like that in `zenith_with_baseline`. Maybe that's enough, or maybe that should be moved/copied to the top of the file.

**Review by hlinnaka — APPROVED** [review]
Great, this will give performance issues a lot more visibiity!

File: test_runner/fixtures/compare_fixtures.py (hunk 1 above)
Comment (bojanserafimov) [reply test_runner/fixtures/compare_fixtures.py:12]: expanded the zenith_with_baseline docstring with intent/instructions, and added pointer to it from the PgCompare docstring.

**Pull request merged**